sunkit_image/coalignment/tests/test_coalignment.py

```diff
@@ -0,0 +1,147 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+import sunpy.map
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+from sunkit_image.coalignment import coalign
+from sunkit_image.coalignment.interface import (
+    REGISTERED_METHODS,
+    AffineParams,
+    _update_fits_wcs_metadata,
+    register_coalignment_method,
+)
+from sunkit_image.tests.helpers import figure_test
+
+
+@pytest.fixture()
+def eis_test_map():
+    url = "https://github.com/sunpy/data/raw/main/sunkit-image/eis_20140108_095727.fe_12_195_119.2c-0.int.fits"
+    with fits.open(url) as hdul:
+        return sunpy.map.Map(hdul[0].data, hdul[0].header)
+
+
+@pytest.fixture()
+def aia193_test_map(eis_test_map):
```

Comment:
How do we feel about this being a file we download from github like the one above?

Reply:
That's a good idea. This download consistently fails.